src/notification.h

```diff
@@ -9,6 +9,12 @@
 
 #define DUNST_NOTIF_MAX_CHARS 5000
 
+enum BehaviorFS {
```

Comment:
> I'd normally drop reviews like this as it's working perfectly and there are no relevant issues with the code but submitted anyway since @bebehei prodded me to.

This is exactly the reason, why I wanted to get a nitpicky review. One of the things, I dislike about dunst is the hard inconsistency of specific things (like typenames). Because even I fail for myself to stick to one schema.